tests/behat/features/paragraph.civictheme_automated_list.render.feature

```diff
@@ -77,12 +77,12 @@ Feature: Automated list render
 
     And I should see a ".ct-list__rows" element
     # 16 items = 15 pages - 2 pages unpublished + 3 events
-    And I should see 16 ".ct-item-grid__item" elements
```

Comment:
`item-grid` was removed in favour of using `ct-grid`, so not binding to items of `item-grid` anymore